tests/testData.ts

```diff
@@ -0,0 +1,36 @@
+export const SoldierToBeAddedForPOSTtest = {
+  _id: '666',
+  name: 'Luke Skywalker',
+  rank: 'Jedi Master',
+  limitations: [
+    'Missing one Hand',
+    'Whiney bratt',
+  ],
+};
+
+export const SoldierToBeAddedForFailedPOSTtest = {
+  _id: '101',
+  rank: 'none',
+  limitations: [
+    'Missing name',
+  ],
+};
+
+export const SoldierToBeFound = {
+  _id: '1',
+  name: 'Test Boi',
+  rank: 'Test',
+  limitations: [
+    'Test Subject: (ᵘᆸᵘ)⭜',
+  ],
+};
+
+export const SoldierToSearch = {
+  _id: '179',
+  name: 'Darth Vader',
+  rank: 'Silver',
+  limitations: [
+    'Useless',
+    'no_one_likes_this_guy',
+  ],
+};
```

Comment:
```TS
export {
  goodSoldier,
  badSoldier,
  searchSoldier,
}
```

Reply:
That way you can import it like this:
```TS
import * as soldiers from './testData'
```


Another option is to create a json file. You can also import json files, and this is only data so I recommend it in this case